lib/column/ipv4_test.go

```diff
@@ -0,0 +1,211 @@
+package column
+
+import (
+	"net"
+	"testing"
+)
+
+const invalidIPv4Str = "44.489.38.222"
+
+func TestIPv4_AppendRow_InvalidIP(t *testing.T) {
+	col := IPv4{}
+
+	// appending string
+	err := col.AppendRow(invalidIPv4Str)
+	if err == nil {
+		t.Fatalf("AppendRow didn't IP")
+	}
+	switch e := err.(type) {
+	case *ColumnConverterError:
+		if e.Hint != "invalid IP format" {
+			t.Fatalf("expected error hint %s, got %s", "AppendRow didn't IP", e.Hint)
+		}
+	default:
+		t.Fatalf("AppendRow didn't IP")
+	}
+}
+
+func TestIPv4_Append_InvalidIP(t *testing.T) {
+	strIps := []string{
+		"43.58.38.33",
+		invalidIPv4Str,
+	}
+
+	// appending strings
+	col := IPv4{}
+	err := col.AppendRow("127.0.0.1") // add 1 valid IP
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	_, err = col.Append(strIps)
+	if err == nil {
+		t.Fatal("Append must return error if invalid IP provided")
```

Comment:
see above using require and assert packages in test.

Reply:
Done